src/mudlet.h

```diff
@@ -391,6 +393,11 @@ private slots:
     HostManager mHostManager;
 
     bool mshowMapAuditErrors;
+    bool mCompactInputLine;
+
+    void slot_toggle_compact_input_line();
```

Comment:
As this is a SLOT does it not have to be listed under either `public slots:` or `private slots:` so the `moc` can find it?

Reply:
Apparently not, it works as-is.